README.md

```diff
@@ -22,7 +22,7 @@ Hopefully this means that if you'd like to use this library, you can by simply d
 pip install basketball_reference_web_scraper
 ```
 
-This library requires `Python 3.4+` and only supports seasons after the `1999-2000` season
+This library requires `Python 3.4+` and only supports seasons after the `1948-1949` season
```

Comment:
I've looked into this and I think increasing the support to include these seasons introduces a couple of problems

1. Technically, the NBA didn't exist in `1948-1949`. The `BAA` _did exist_ so need to change the URL to scrape for that season (https://www.basketball-reference.com/leagues/BAA_1949.html)

<img width="816" alt="image" src="https://user-images.githubusercontent.com/8136030/79614817-5a500280-80cf-11ea-99df-f0774afcb30e.png">)

_or_ add the ability to scrape for a given league - which means starting to add a `League` enum for league-based support everywhere (probably defaulting to `League.NBA` in most places).

Secondly, you'll notice the HTML tables look a lot different than the ones for more recent season's (here's this current season's schedule, for example)

<img width="849" alt="image" src="https://user-images.githubusercontent.com/8136030/79614999-b024aa80-80cf-11ea-9c42-93eff5e0e605.png">

There will need to be some adjustments to the schedule parsing logic to handle the lack of hour / minute specificity.

Overall, I think increasing full support back to `1948` will require some rethinking / retooling before being released successfully.

Reply:
Thanks for the feedback! My additions were actually meant to work for the `1950` season onward. Sorry if that wasn't clear. So I think the URL should work fine. But your point about the HTML still stands.

I think the only real differences in the HTML are that:

1. There's no column for start time until the `1976` season.

2. Attendance wasn't recorded until the `1980` season and not consistently until the `1981` season. But your library doesn't include attendance.

Do you think it would be a good idea to just replace the missing values with zeros? Then the users hopefully will be able to tell that it wasn't recorded. So for example, `start_time` would be `datetime.datetime(1950, 11, 1, 0, 0)` for November 1st 1950.

Reply:
> Do you think it would be a good idea to just replace the missing values with zeros? Then the users hopefully will be able to tell that it wasn't recorded.

This could work, though I might just return a `date` object instead, or add a `start_date` property on the returned object that references the correct date but then `None` for `start_time`. This way, the most accurate data is returned while still supporting this date / date time discrepancy.